src/main/java/com/revlearn/team1/dto/TransactionDTO.java

```diff
@@ -0,0 +1,11 @@
+package com.revlearn.team1.dto;
+
```

Comment:
can you create separate DTOs for request and response, so we have the flexibility to change them both separately

Reply:
This PR will auto-update with any changes you push to this branch!